plugins/notify.py

```diff
@@ -9,3 +9,19 @@ def send_notification(self, build_id, text):
         notification_list = self.load("notify_{}".format(build_id), '')
         self.say("{} BuildID: {}, Message: {}".format(notification_list, build_id, text)) 
 
+    @respond_to("^subscribe (@*)(?P<user>\w*) to (?P<build_id>\w*)")
+    def subscribe(self, message, user, build_id):
+        """
+        subscribe [user] to [jobid]: request to be notified when builds complete
+        """
+        if user == "me":
+            user = message.sender.nick
+        else:
+            user = user.replace("@", "")
+        notification_list = "{} @{}".format(
+                self.load("notify_{}".format(build_id), ''), 
+                user)
```

Comment:
`notification_list = "{} @{}".format(self.load("notify_" + build_id, ''), user)` would be clearer.